packages/node/src/yargs.ts

```diff
@@ -22,6 +25,28 @@ export const yargsOptions = yargs(hideBin(process.argv))
       return forceCleanInit();
     },
   })
+  .command({
+    command: 'reindex',
+    describe:
+      'Reindex to specified block height (Once the command is executed, the application would exit upon completion).',
```

Comment:
@deverka same here.

I think we also need to describe that historical is required for this function.